src/apps/beefy/positions.ts

```diff
@@ -276,6 +280,7 @@ const beefyGovVaultsPositions = async (
     functionName: 'getUserClmPools',
     args: [
       address,
+      // @ts-expect-error tokenAddress from beefy response is not always defined
```

Comment:
@jeanregisser with the changes tried a few different ways to fix this error and was unsuccessful.

Reply:
Could/should we filter these to only those with a `tokenAddress` set?

Reply:
Fixed in d47baf4220bf1278b42ebc120ec9a18cd8b8364f!